internal/operators/manager.go

```diff
@@ -193,6 +199,23 @@ func (mgr *Manager) UpdateDependencies(cluster *common.Cluster) error {
 	return nil
 }
 
+// GetSupportedOperators returns a list of OLM operators that are supported
+func (mgr *Manager) GetSupportedOperators() []string {
+	keys := make([]string, 0, len(mgr.olmOperators))
+	for k := range mgr.olmOperators {
+		keys = append(keys, k)
+	}
+	return keys
+}
+
```

Comment:
I understood this API endpoint as returning all the supported Operators (matching `GetMonitoredOperatorsList`). In case it's only OLM-related since we do not need to show the builtins to the user, let's change the endpoint&function to use the name OLM `/supported-olm-operators` and `GetSupportedOLMOperators` accordingly.

Although, maybe it could be useful for future use to be able to even configure builtin operators, if we found it useful.
WDYT? @avishayt

Reply:
As the user doesn't need to know anything about different types of operators, from their perspective it's just "operators" and as a result "supported-operators" seems OK. 
If there is a need in the future to return other types of operators, this endpoint might get an optional parameter instructing what type of operators should be returned.

Reply:
Agree with @jakub-dzon - these are any operators that are available for the user to configure.  I think it's fine as is.